apps/theming-designer/src/components/SamplesCardHeader.scss

```diff
@@ -0,0 +1,5 @@
+:global {
+  .samplescardheader {
+    border-bottom: 1px black solid;
```

Comment:
Might want to add `text-transform: uppercase;` to keep in line with the styling, and not require callers to know that things should be upcased.